### DISCUSSION TITLE
Auth Timeouts

### DISCUSSION THREAD

**Discussion by GabeRoze** [supabase/supabase#22298] [Questions]
Is anyone experiencing auth timeouts? I am suddenly unable to authenticate with OTP today Wed Mar 27. I have not made changes to my codebase.

**Comment by GaryAustin1** [comment]
Your posts here and Discord are the only ones mentioning this.  Any errors in auth logs?

**Reply by GabeRoze** [reply]
@GaryAustin1 marked as duplicate. Another post covers the issue. My logs showed either no errors or a few 500 errors

[supabase_logs.csv](https://github.com/supabase/supabase/files/14781608/supabase_logs.csv)

**Reply by GaryAustin1** [reply]
There have been issues with gmail SMTP the past few days.  I thought users were mainly getting errors though, and you did not mention emails not going thru so did not connect the two.

**Reply by GabeRoze** [reply]
Ah gotcha. It took me a bit of digging until I connected them myself. As you're a maintainer, would you recommend using some other relay service?

**Reply by GaryAustin1** [reply]
I use Brevo myself.  The gmail thing is odd as it does not appear a global issue, but the other SMTP providers all seem to be working.

**Reply by GabeRoze** [reply]
Thanks! I'll check it out.

**Comment by GabeRoze** [comment]
Duplicate of https://github.com/supabase/supabase/issues/22296
